sdk/appconfiguration/Azure.Data.AppConfiguration/src/ConfigurationClient.cs

```diff
@@ -139,6 +141,8 @@ public virtual Response<ConfigurationSetting> Add(ConfigurationSetting setting,
                     case 200:
                     case 201:
                         return CreateResponse(response);
+                    case 412:
```

Comment:
What is it that makes it so you can't build the sync version of Add using the AddAsync.Result or similar? I only noticed it because you had to add in the same error handling twice.